### PR TITLE
LL-5422 (Announcements): add device firmware + platform filtering

### PR DESCRIPTION
**Pull request by LFBarreto**
(Announcements): add device firmware + platform filtering

### Type

Feature
### Context

[LL-5422]
### Parts of the app affected / Test plan

Announcements + debug mock announcements

[LL-5422]: https://ledgerhq.atlassian.net/browse/LL-5422

### CODE REVIEW

**Review by valpinkman — DISMISSED** [review]

File: src/renderer/components/AnnouncementProviderWrapper.js [hunk 1]
```diff
@@ -67,6 +68,8 @@ export function AnnouncementProviderWrapper({ children }: Props) {
     language,
     currencies,
     getDate: () => new Date(),
+    lastSeenDevice: lastSeenDevice || undefined,
+    platform: process.platform,
```
Comment (gre) [inline src/renderer/components/AnnouncementProviderWrapper.js:72]: process.platform is one of 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32' ( https://nodejs.org/api/process.html#process_process_platform )

which is not what you seem to expect at
https://github.com/LedgerHQ/ledger-live-common/blob/84d9acd711476c0ab95294f29918ad22bc87468c/src/notifications/AnnouncementProvider/logic.js#L22

**Review by gre — CHANGES_REQUESTED** [review]
i don't see how the platform field can work here. **Was this tested?**

**Review by imqdee — DISMISSED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by Arnaud97234 — DISMISSED** [review]
Tested on Mac, Linux and Windows.
✅

**Review by Arnaud97234 — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Review by nabil-brn — APPROVED** [review]
Testing done by Arnaud.

**Pull request merged**